src/test/Components/Blueprints/Blueprints.test.js

```diff
@@ -55,8 +55,14 @@ describe('Blueprints', () => {
       )
     );
 
+    // const { router } = await renderWithReduxRouter('', {});
```

Comment:
leftover?

Reply:
oh yeah, I meant to fix the test to actually test the button works - found the fix :)